src/sentry/testutils/helpers/backups.py

```diff
@@ -150,7 +155,7 @@ def import_export_then_validate(method_name: str, *, reset_pks: bool = True) ->
 
         # Validate that the "expected" and "actual" JSON matches.
         actual = export_to_file(tmp_actual)
-        res = validate(expect, actual)
+        res = validate(expect, actual, get_default_comparators())
```

Comment:
Technically the 3rd arg is not needed, right?

```suggestion
        res = validate(expect, actual)
```